doc/index.rst

```diff
@@ -59,20 +59,30 @@ QGreenland Documentation
     \part{Reference}
 
 
-.. toctree::
-    :name: Reference
-    :caption: Reference
-    :maxdepth: 1
-    :glob:
-    :hidden:
-
-    reference/glossary/index
-    reference/data-compatibility-guide.md
-    reference/online-resources.md
-    reference/architecture/index
-    reference/api/index
-    reference/cli/index
-    reference/*
+.. only:: latex
+
+  .. toctree::
```

Comment:
In this configuration, the User Reference will not show in HTML output on ReadTheDocs. We _do_ want it to show in both right? I think we should extract this from `.. only:: latex`` conditional, and always include it.

Reply:
If we do indeed need this `.. only:: latex` directive to get our desired behavior, I totally don't understand why it behaves as it does. If that's the case, let's add a comment describing what this does and that it was intended.